pyvlx/discovery.py

```diff
@@ -41,7 +41,7 @@ def add_info_and_host(fut: Future) -> None:
             info: AsyncServiceInfo = fut.result()
             self.infos.append(info)
             host = VeluxHost(
-                hostname=info.name.replace("._http._tcp.local.", ""),
+                hostname=info.name.replace("._http._tcp.local.", "").upper().replace("LAN_", ""),   # KLF report it's hostname within DHCP requests without the "LAN_" prefix
```

Comment:
Im fine with this hack - May i ask you to do this in in a separate function:

```
def sanitize_hostname(hostname):
    hostname=hostname.upper()
    hostname=hostname.replace("._HTTP._TCP.LOCAL.", "")
    hostname=hostname.replace("LAN_", "")
```
and `hostname=sanitize_hostname(info.name)`

and then `hostname=hostname` within the parameter...